### ISSUE TITLE
Do contributors own the copyright to their own commits?

### ISSUE THREAD

**Issue by L1lith** [melonjs/melonjs.org#14]
I noticed the site and the README said the site was copyrighted, but I was wondering if that means stuff we commit is copyrighted too or if there should be some kind of "contribution guidelines" that that could outline that kinda stuff. Not a big deal (I don't have any intention of copying the site or making a lawsuit) but it seems like a good thing to figure out.

**Comment by obiot** [comment]
yeah the copyright is just me actually (AltByte is my company here), but we could also add either a changelog or a contributor list (although the latter is redundant with Github as it already displays it)
